src/content/reference/app-defaulting-validation/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Defaulting and Validation of app CRs
+description: How defaulting and validation of app CRs is implemented by app-admission-controller
+weight: 110
+type: page
+last_review_date: 2020-12-16
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+---
+
+## Overview
+
+For Giant Swarm releases using app-operator version 3.0.0 and upwards the
+defaulting and validation logic of [app CRs]({{< relref "/content/reference/cp-k8s-api/apps.application.giantswarm.io.md" >}}) is enabled. This logic is provided by
+[app-admission-controller](https://github.com/giantswarm/app-admission-controller).
+
+We have not enabled the defaulting and validation for existing app CRs to avoid
+disrupting your existing usage of App Platform.
+
+## Defaulting
+
+The defaulting logic is implemented using a mutating webhook. It currently
+defaults the following settings in the app CR if they are not specified.
+
+- The `app-operator.giantswarm.io/version` label that determines which instance of app-operator processes the app CR.
+- The kubeconfig secret that lets app-operator connect to your cluster.
+- The cluster values configmap that contains per cluster values like the base
+domain for your cluster.
+
+Here is an example app CR with only the settings you need to provide. The user
+values configmap is optional, if you wish to configure the app with your own values.
+
+```yaml
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  name: "my-kong"
+  namespace: "x7jwz"
+spec:
+  catalog: "giantswarm"
+  name: "kong-app"
+  namespace: "kong"
+  version: "0.7.2"
+  userConfig:
+    configMap:
+      name: "kong-user-values"
+      namespace: "x7jwz"
+```
```

Comment:
IMO both YAML snippets could be easier to read if the string quoting would be removed.

Reply:
Much nicer. I'll get rid of the quotes.